### ISSUE TITLE
"No log" when training RobertaForSequenceClassification using Trainer

### ISSUE THREAD

**Issue by BryanWBear** [huggingface/transformers#8910]
When training, for the first few logging steps I get "No log".

Looks like this:

Step | Training Loss | Validation Loss | Accuracy | F1
-- | -- | -- | -- | --
150 | No log | 0.695841 | 0.503277 | 0.410575
300 | No log | 0.696622 | 0.488860 | 0.298561
450 | No log | 0.694300 | 0.499345 | 0.356902

What does this mean? My classifier is performing poorly and I am wondering if this is related. I am finetuning roberta-base using 3k question-answer pairs, 50% positively labelled, 50% negatively labelled.

Thanks,
Bryan

**Comment by LysandreJik** [comment]
Could you provide the environment information as mentioned in the issue template, alongside the a reproducible that outputs this so that we may check what's going on? Thank you.

**Comment by denocris** [comment]
Hi @BryanWBear , 

I am facing this issue too. In the meantime, did you find a solution?

Thank you so much in advance!

**Comment by conraddd** [comment]
the default `logging_steps` in `TrainingArguments` is set to `500` steps, so no loss is reported before 500 steps
